docs/consignments.md

```diff
@@ -1,33 +1,78 @@
 # Consignment configuration
 
-The consignments can be either purely synthetic or based on AQAS inspection
- records (Form 280 or AQIM). Configuration for the consignments is under
- the `consignment` key in the configuration file.
+## Consignment generation method
+The consignments can be either purely synthetic or based on inspection records
+ (e.g., Form 280 or AQIM). Configuration for the consignments is under the
+ `consignment` key in the configuration file.
+
+ The `generation_method` key can be either `parameter_based` to generate
+ synthetic consignments based on user-provided list of parameter values, or
+ `input_file` to create consignments that match inspection records in a CSV
+ file.
```

Comment:
The leading space coming from the original text is just a typo as far as I can tell.